README.md

```diff
@@ -12,6 +12,13 @@ Beautiful, accessible drag and drop for lists with [`React.js`](https://facebook
 
 See how beautiful it is for yourself!
 
+## Read this in other languages
```

Comment:
can you please move this new section below the examples section (originally line 26)

Reply:
Ok